src/python/grapl_analyzerlib/mypy.ini

```diff
@@ -1,10 +1,10 @@
 [mypy]
-python_version = 3.8
+python_version = 3.10
```

Comment:
I'm not sure this file is even used anymore 🤔

(This can be ironed out outside of this PR, though.)